modin/core/execution/ray/common/utils.py

```diff
@@ -217,7 +217,17 @@ def initialize_ray(
         _move_stdlib_ahead_of_site_packages
     )
     ray.worker.global_worker.run_function_on_all_workers(_import_pandas)
-    num_cpus = int(ray.cluster_resources()["CPU"])
+    num_cpus = ray.cluster_resources().get("CPU", None)
+    if num_cpus is None:
+        warnings.warn(
+            "The current Ray cluster does not have any CPU Resources.\nModin uses the number of "
+            + "CPUs to determine how many partitions to create.\nNumber of partitions defaulting to"
+            + " number of CPUs on head node. To update, run the following python code:\n\tfrom "
```

Comment:
nit: could you instead instantiate `num_cpus` above this warning, and add the actual value of `num_cpus` to the warning message? such as, with an f-string.